app/assets/javascripts/application.js

```diff
@@ -123,6 +123,7 @@ function onContentLoad(){
       if ($(this).data('on-complete')){
         window[$(this).data('on-complete')].call(null, this, status);
       }
+      $(document).trigger('ContentLoad');
```

Comment:
Why is this needed? it will cause the `onContentLoad` function to run again for every ajax-url. If you need to handle some initialization (i.e. datatables) once the content loads, it would be better to add that in the data-on-complete attribute which is called once the loading completes.